Added some faqs for space race. (#355)
filecoin-project/filecoin-docs: docs/mine/spacerace.md

- If the deal is marked as successful on the dashboard but the above commands do not return the expected result, simply repeat steps #1-3 above.

#### How do I force a WaitDeals sector to start sealing?

Run `lotus-miner sectors seal <sector id>`.

> #### How is the deal rate of the combined miners being calculated?

Reviewer: Let's remove as a standalone FAQ. I added a commit to line 112 that clarifies this, and we can point miners to that in the future.

The deal rate is calculated based on the sum of all the deals across the miners.

#### If I rent a server in EU from US, can I compete in EU region?

> The definition of the `mining operation` can be found [here](https://spacerace.filecoin.io/rules), in short, it is the physical location of the storage and sealing hardware. It might not be cost effective, but you can rent cloud service anywhere in the world. However, it is NOT acceptable to do something like running a proxy or anything of that nature.

Reviewer: ```suggestion
Your region is determined by the physical location of the storage and sealing hardware. It might not be cost effective, but yes, it is permissible to run your mining operation on a rented service anywhere in the world. However, it is NOT acceptable to obscure your miner location via a proxy or similar service."
```

Remove link to website with rules, since this site gets updated more frequently.

Reviewer: Can you also move this FAQ to immediately below "How is the "location" of a mining operation determined?"

#### Why is my deal success rate below 80%?

> Deals can fail due to various reason. Go to [Space Race Dashboard](https://spacerace.filecoin.io/), search for your miner id and click `Open Logs` to see the errors that caused the failures.

Reviewer: ```suggestion
Deals can fail due to various reasons. To see the deal logs for your miner, go to [Space Race Dashboard](https://spacerace.filecoin.io/), search for your miner id, and click `Open Logs`. See the [Improving Connectivity > Common Errors](https://docs.filecoin.io/mine/connectivity/#common-errors) section for detailed explanations and recommended fixes for each error.
```

Give users the next logical step to fix their problem.

> #### My sector is stuck in X stage, how can I remove it?

Reviewer: ```suggestion
#### My sector is stuck in X stage. How can I remove it?
```

> After Lotus version v0.5.6, you can remove a sector by running `lotus-miner sectors remove --really-do-it <sector id>`. Warning: by doing this means losing power and collateral for the removed sector.

Reviewer: ```suggestion
In lotus v0.5.6 and later, you can remove a sector by running `lotus-miner sectors remove --really-do-it <sector id>`. Warning: This will cause you to lose all power and collateral for the removed sector.
```
